pkg/cmd/cli/cmd/idle.go

```diff
@@ -532,7 +532,7 @@ func (o *IdleOptions) RunIdle(f *clientcmd.Factory) error {
 	}
 
 	delegScaleGetter := osclient.NewDelegatingScaleNamespacer(oclient, kclient)
-	dcGetter := deployclient.New(oclient.RESTClient)
+	dcGetter := deployclient.New(oclient)
```

Comment:
@juanvallejo I investigated this and realized that needs to rebase with upstream kubernetes, because this is auto-generated based on this template:

https://github.com/kubernetes/kubernetes/blob/master/cmd/libs/go2idl/client-gen/generators/generator_for_clientset.go#L191